Madness/FlatMap.swift

```diff
@@ -9,6 +9,13 @@ public func >>- <C: CollectionType, T, U> (parser: Parser<C, T>.Function, f: T -
 	}
 }
 
+/// Returns a parser which always ignores its input and produces a constant value.
+///
+/// When combining parsers with `>>-`, allows constant values to be injected into the parser chain.
+public func inject<C: CollectionType, T>(value: T) -> Parser<C, T>.Function {
```

Comment:
What do you think about `pure` for this? I’m not particularly married to `unit` as used in Prelude, and `inject` always reminds me of `reduce` (I can thank Smalltalk for that).